srcf/controllib/emails/member/remove-admin.txt

```diff
@@ -0,0 +1,3 @@
+You have been removed from the '{{ society.society }}' shared account on the SRCF.
```

Comment:
Was this one just a missing file?

Reply:
See 485228c -- previously we sent the _$user removed from $group_ email to the pre-remove list of admins (i.e. including the user just removed), but this was refactored a while ago to send emails based on a `Society` object rather than a list of names and addresses.

`recipients` is dead code, and the removed admin doesn't currently get notified -- this is a new email just for them.